pkg/processor/runtime/rpc/connection/abstract.go

```diff
@@ -88,8 +94,20 @@ func (bc *AbstractConnectionManager) UpdateStatistics(durationSec float64) {
 	bc.Configuration.Statistics.DurationMilliSecondsSum += uint64(durationSec * 1000)
 }
 
+func (bc *AbstractConnectionManager) GetConfig() ManagerConfigration {
+	return *bc.Configuration
+}
+
+func (bc *AbstractConnectionManager) IsBusy() bool {
+	return len(bc.allocator.GetObjects()) == bc.allocator.GetNumObjectsAvailable()
+}
```

Comment:
Isn't it the other-way around?
It is busy when there are no objects available.
When all objects are in the channel - they are available for allocation.

Reply:
@TomerShor the purpose of isBusy is to signal whether at least one event is being processed now by manager. I see now that the name might be confusing. Maybe `InUse` would be better?

Reply:
Oh then I suppose `IsBusy` is ok too, I thought it meant it can't take any more events.
but there's still an issue here:
```suggestion
func (bc *AbstractConnectionManager) IsBusy() bool {
	return len(bc.allocator.GetObjects()) != bc.allocator.GetNumObjectsAvailable()
}
```